backend/app/repositories/base.py

```
@@ -0,0 +1,69 @@
+"""Base repository for CRUD operations."""
+
+from typing import Any, Dict, Generic, List, Optional, Type, TypeVar
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)